iree/samples/simple_embedding/CMakeLists.txt

```diff
@@ -53,64 +34,99 @@ iree_cc_binary(
 
 iree_cc_binary(
   NAME
-    simple_embedding_vulkan
+    simple_embedding_local_sync
   SRCS
-    "device_vulkan.c"
+    "device_local_sync.c"
     "simple_embedding.c"
   DEPS
-    ::simple_embedding_test_bytecode_module_c
-    ::simple_embedding_test_llvm_aot_rv64
+    ::simple_embedding_test_bytecode_module_dylib_riscv_64_c
+    ::simple_embedding_test_bytecode_module_dylib_x86_64_c
     iree::base
     iree::hal
-    iree::hal::vulkan::registration
+    iree::hal::local
+    iree::hal::local::loaders::embedded_library_loader
+    iree::hal::local::sync_driver
     iree::modules::hal
     iree::vm
     iree::vm::bytecode_module
 )
 
+iree_bytecode_module(
```

Comment:
it'd be nice to have aarch64 here too so we have some kind of android coverage on the CIs

Reply:
Module added. However, Android currently doesn't build sample projects (or test them). We can address it later.